src/azul/__init__.py

```diff
@@ -881,18 +881,24 @@ def api_gateway_lambda_timeout(self) -> int:
     # This is set dynamically at runtime
     lambda_context: Optional[LambdaContext] = None
 
+    @property
+    def _timing_is_restricted(self):
```

Comment:
Would benefit from a return type annotation

```suggestion
    def _timing_is_restricted(self) -> bool:
```